packages/core/content-releases/shared/contracts/releases.ts

```diff
@@ -18,6 +18,12 @@ export type Pagination = {
 
 export interface ReleaseDataResponse extends Omit<Release, 'actions'> {
   actions: { meta: { count: number } };
+  createdBy: {
+    id: string;
+    firstname: string;
+    lastname: string;
+    username: string;
+  };
```

Comment:
You shouldn't need to do this. Release extends Entity which does this

https://github.com/strapi/strapi/blob/30e8a6321d5a19ab8b01c0912be65f4782667acd/packages/core/content-releases/shared/types.ts#L22-L26